### DISCUSSION TITLE
Facing issue when trying to use pagination with inner_join

### DISCUSSION THREAD

**Discussion by vlasfama** [diesel-rs/diesel#3010] [Q&A]
### Versions

- **Rust: 1.58.0-nightly**
- **Diesel: version:1.4.7**
- **Database: Postgres**
- Operating System: macOS Monterey 

### Feature Flags

- **diesel:  "postgres","chrono","numeric","uuidv07","r2d2",**

## Problem Description
`'Problem : DatabaseError(__Unknown, "column reference \"created_at\" is ambiguous")'`
`'Problem : DatabaseError(__Unknown, "column reference \"id\" is ambiguous")'`

### What are you trying to accomplish?
I am trying to do inner_join and pagination to my table, join working fine, but when I am trying to use pagination, the query gives me above mention error.

### What is the expected output?
I am trying to use the  inner join and pagination 

```
let mut q = Organistaion::table.inner_join(Employee::table.on(Organistaion::dsl::employee_id.eq(Employee::dsl::user_id)),)
            .into_boxed();
let conn = self.pool.get()?;
let query = q.paginate(Organistaion::dsl::created_at, &params)?; // &param contains filter query. 
let res = q.load::<(OrganisationInfo, EmployeeInfo)>(&conn);
```
### What is the actual output?
I would like to use an `alias` to the above query below is raw `sql_query` 
```
SELECT *, COUNT(*) OVER () FROM (
SELECT "organisation"."id", "organisation"."name", "organisation"."employee_id","employee"."id", 
"employee"."user_id", "employee"."mobile", "employee"."location", "employee"."created_at" 
FROM ("organisation" INNER JOIN "employee" ON "organisation"."employee_id" = "employee"."user_id") 
WHERE "employee"."user_id" = $1 AND "organisation"."name" = $2 AND "organisation"."name" = $3 and ("organisation"."created_at", "organisation"."id") >= ($4, $5)  ORDER BY "organisation"."created_at"
) as t LIMIT $6
```
## Checklist

- [ ] I have already looked over the [issue tracker](https://github.com/diesel-rs/diesel/issues)  and the [disussion forum](https://github.com/diesel-rs/diesel/discussions)  for similar possible closed issues.
 
- [ ] This issue can be reproduced on Rust's stable channel. (Your issue will be
  closed if this is not the case)
- [ ] This issue can be reproduced without requiring a third party crate

**Comment by weiznich** [comment]
Our issue tracker is no support forum. Please use the discussion part of the repository to ask questions. (I've moved this question for now)

Other than that you may want to include the following relevant information as otherwise it's not possible to help you here:
* Any type and function definition that is not coming from diesel or rust-std lib. 
* Any relevant trait implementations.
* The definition of `.paginate`
* Any information releated to the `params` variable.
